.github/workflows/rust.yml

```diff
@@ -30,6 +30,27 @@ jobs:
   build_and_test:
     name: Rust and clippy tests
     runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        sed:
+          - GNU
+        bash_version:
+          - "5.2"
+          - "5.1"
+          - "5.0"
+          - "4.4"
+          - "4.3"
+          - "4.2"
+          - "4.1"
+          - "4.0"
+          - "3.2"
+          - "3.1"
+          - "3.0"
```

Comment:
The interface on GitHub is confusing.

![image](https://github.com/user-attachments/assets/b7c369ca-8e02-4a88-a01d-a3b3ff2b716c)

`GNU sed, bash x.x` is desirable. Should be improved